### PR TITLE
[ENDPOINT] Endpoint to list contacts OT246-63

### PR DESCRIPTION
**Pull request by Samuel0023**
### Description
AS user
I WANT to list the contacts of the page
TO be able to see all that exist

Criteria of acceptance:
GET /contacts - Will return all contact records. This endpoint can only be accessed by an administrator user

### Evidence 
with one contact:
image

without contacts:
image

### CODE REVIEW

**Review by gabodmc — APPROVED** [review]

File: routes/contacts.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { Router } = require('express')
+const { get } = require('../controllers/contacts')
+const { authUser } = require('../middlewares/authUser')
+
+const router = Router()
+
+router.get('/', authUser, get)
```
Comment (ghnoob) [inline routes/contacts.js]: metele el middle de admin tambien, que ya esta hecho

File: services/contacts.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const { Contact } = require('../database/models')
+const { ErrorObject } = require('../helpers/error')
+
+exports.getContacts = async () => {
+  try {
+    const contacts = await Contact.findAll()
+    if (contacts.length === 0) {
+      throw new ErrorObject('No contacts found', 404)
+    }
```
Comment (ghnoob) [inline services/contacts.js]: cuando se hace un get sin id no se manda un 404, se devuelve un array vacio

**Review by ghnoob — CHANGES_REQUESTED** [review]

File: routes/contacts.js (hunk 1 above)
Comment (Samuel0023) [reply routes/contacts.js]: dale 👍🏽

File: services/contacts.js (hunk 2 above)
Comment (Samuel0023) [reply services/contacts.js]: ty 👍🏽

**Review by bgalacho — APPROVED** [review]
Crack

**Review by ghnoob — APPROVED** [review]
lindo trabajo

**Pull request merged**